### DISCUSSION TITLE
How to use the launch method to pass data to new screen

### DISCUSSION THREAD

**Discussion by segungreat** [bhoominn/nb_utils#19] [General]
I have something like this, 
`ICChartScreen().launch( context,
     pageRouteAnimation: PageRouteAnimation.Slide
 );`

and i want to pass data to the next screen (ICChartScreen)

Am unable to share data between screen

**Comment by bhoominn** [comment]
You can create constuctor to pass the data like this.
`ICChartScreen(key: data).launch( context, pageRouteAnimation: PageRouteAnimation.Slide );`
